components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -118,10 +118,10 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
     private var discardAutoCompleteResult: Boolean = false
 
     val nonAutocompleteText: String
-        get() = getNonAutocompleteText(text)
+        get() = getNonAutocompleteText(text!!)
```

Comment:
What do you think about adding an override for `getText`? Then, at least, we don't need to add those `!!` everywhere. I understand text can only be null for API < 28 anyway.

```
override fun getText(): Editable {
        return super.getText() as Editable
}
```